actors/miner/src/lib.rs

```diff
@@ -2299,6 +2299,8 @@ impl Actor {
             // NOTE: This isn't quite correct. If we repeatedly fill, empty,
             // fill, and empty, the queue, we'll keep scheduling new cron
             // jobs. However, in practice, that shouldn't be all that bad.
+
+            // TODO: immediately terminate the deals
```

Comment:
I don't think there's anything to do here. The market is notified in process_early_terminations.